app/scripts/HiGlassComponent.js

```diff
@@ -3255,16 +3296,28 @@ class HiGlassComponent extends React.Component {
       relTrackY: (hoveredTrack && hoveredTrack.flipText) ? relTrackPos[0] : relTrackPos[1],
       dataX,
       dataY,
-      isFrom2dTrack: hoveredTrack && hoveredTrack.is2d,
-      isFromVerticalTrack: hoveredTrack && hoveredTrack.flipText,
+      isFrom2dTrack: !!(hoveredTrack && hoveredTrack.is2d),
+      isFromVerticalTrack: !!(hoveredTrack && hoveredTrack.flipText),
       track: hoveredTrack,
       origEvt: e,
       sourceUid: this.uid,
       hoveredTracks,
+      noHoveredTracks: hoveredTracks.length === 0,
```

Comment:
See my comment on `noHoveredTracks` further down. I think it's unnecessary because we can just use `hoveredTracks.length === 0` which is just as clear and doesn't require having another variable to keep track of.

Reply:
See my comment above. I believe it's necessary because `hoveredTracks` is removed from global mouseMove events.